### DISCUSSION TITLE
LYWSD03MMC Which firmware to choose for BLE

### DISCUSSION THREAD

**Discussion by Skol6** [pvvx/ATC_MiThermometer#475] [Q&A] [answered]
Hi,
sorry to ask, but I do not get which firmware I need to choose to use only bluetooth.
For zigbee I need to buy an adapter 1st.

Currently I am using the firmware from: [https://github.com/atc1441/ATC_MiThermometer](https://github.com/atc1441/ATC_MiThermometer)
Or do I have to revert to standard first?
Thanks

**Comment by pvvx** [comment, marked as answer]
Use https://atc1441.github.io/TelinkFlasher.html and in the file selection menu enter the text:
```
https://github.com/pvvx/ATC_MiThermometer/raw/master/ATC_v46.bin
```

For settings after flashing, use (https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html).
After flashing, when there is an update, the file selection will be automatically presented in [TelinkMiFlasher.html](https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html).

**Comment by Skol6** [comment]
worked perfectly!
Thanks a lot!
